From a7dfb006872ee6bff19044e0321eabed4910c0ff Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@gnome.org>
Date: Mon, 16 Mar 2015 15:55:27 -0500
Subject: [PATCH] Fix gcab installation

gcab installs a libtool wrapper script unless fast install is enabled.
Odd.

Also, remove the git gen stuff so that autogen from tarball works.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 748e804..65a68b4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_PREREQ(2.57)
-AC_INIT([gcab], m4_esyscmd([build-aux/git-version-gen .tarball-version]), [marcandre.lureau@gmail.com])
+AC_INIT([gcab], [0.5], [marcandre.lureau@gmail.com])
 AC_CONFIG_SRCDIR([gcab.c])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
@@ -12,7 +12,7 @@ AM_SILENT_RULES([yes])
 AC_PROG_CC
 AM_PROG_CC_C_O
 AC_PROG_INSTALL
-LT_INIT([win32-dll disable-fast-install])
+LT_INIT([win32-dll])
 GTK_DOC_CHECK([1.14],[--flavour no-tmpl])
 PKG_PROG_PKG_CONFIG
 GOBJECT_INTROSPECTION_CHECK([0.9.4])
--
2.1.0
